index.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+
+  extends: ["plugin:react/recommended", "prettier/@typescript-eslint", "plugin:prettier/recommended", "prettier/react"],
+
+  plugins: ["simple-import-sort", "jest", "@typescript-eslint", "react-hooks", "react", "prettier", "import"],
+
+  ignorePatterns: ["coverage", "**/dist", ".cache-loader", "node_modules", ".yalc"],
+
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+
+  env: {
+    browser: true,
+    node: true,
+    jest: true,
+    es6: true,
+  },
+
+  rules: {
+    "prettier/prettier": "error",
+
+    // Base
+
+    "no-unused-expressions": ["error", { allowTernary: true }],
+    "no-console": "off",
+    "no-confusing-arrow": "off",
+    "no-else-return": "off",
+    "no-return-assign": ["error", "except-parens"],
+    "no-underscore-dangle": "off",
+    camelcase: "off",
```

Comment:
Any reason we disabled it?

Reply:
I guess we could use `properties: never`? Event properties are `snake_case`, for example, so can't expect to be 100% camelCase everywhere.

Reply:
Yeah, sounds okay!

Reply:
```suggestion
    camelcase: ["error", { properties: "never" }],
```